### PR TITLE
refactor(dropdown): update styles

### PR DESCRIPTION
**Pull request by armandoStratio**
### Documentation
- [ ] Add the issue in PR description
- [x] Have changelog updated
- [ ] You fill the milestone value
- [ ] You add some label
- [x] Have example running in demo app
- [x] Review id on demo is the same of egeo folder

### Code
- [ ] Pass Sass lint task
- [x] Have qaTags

### Tests
- [x] Have at least 70% tests coverage

### CODE REVIEW

File: src/lib/st-dropdown-menu/st-dropdown-menu-item/st-dropdown-menu-item.component.scss [hunk 1]
```diff
@@ -8,19 +8,3 @@
  *
  * SPDX-License-Identifier: Apache-2.0.
  */
```
Comment (jcnunez-st) [inline src/lib/st-dropdown-menu/st-dropdown-menu-item/st-dropdown-menu-item.component.scss]: Remove this file? now it's empty

File: src/lib/st-dropdown-menu/st-dropdown-menu.component.scss [hunk 2]
```diff
@@ -8,38 +8,4 @@
  *
  * SPDX-License-Identifier: Apache-2.0.
  */
```
Comment (jcnunez-st) [inline src/lib/st-dropdown-menu/st-dropdown-menu.component.scss]: remove this file? now it's empty

File: src/lib/st-dropdown/st-dropdown.component.scss [hunk 3]
```diff
@@ -8,49 +8,3 @@
  *
  * SPDX-License-Identifier: Apache-2.0.
  */
```
Comment (jcnunez-st) [inline src/lib/st-dropdown/st-dropdown.component.scss]: remove this file? now it's empty

File: src/egeo-demo/st-dropdown-menu-demo/st-dropdown-menu-demo.html [hunk 4]
```diff
@@ -10,9 +10,19 @@
     SPDX-License-Identifier: Apache-2.0.
 
 -->
-<section class="container-liquid">
-   <st-dropdown-menu [items]="menu" [active]="active" [placement]="'top-start'" (change)="onChange($event)">
-      <button (click)="changeActive()" class="button">Show</button>
+<section
```
Comment (calcerrada) [inline src/egeo-demo/st-dropdown-menu-demo/st-dropdown-menu-demo.html]: Section tag is not a good choice for this case.

File: src/theme/components/_dropdown.scss [hunk 5]
```diff
@@ -8,22 +8,105 @@
...
 
+
+
+
+
+
+
+
+/*
+.st-dropdown-menu {
```
Comment (calcerrada) [inline src/theme/components/_dropdown.scss]: Remove commented code, please

File: src/egeo-demo/st-dropdown-demo/st-dropdown-demo.html [hunk 6]
```diff
@@ -36,41 +47,43 @@
...
       </div>
    </div>
 
    <div class="row">
-      <div class="col-md-24 background-grey">
+      <div class="col-md-12 background-grey">
          <st-dropdown button="Example With Theme Light" themeClass="themeB" [items]="items"></st-dropdown>
       </div>
    </div>
+<!--
```
Comment (jcnunez-st) [inline src/egeo-demo/st-dropdown-demo/st-dropdown-demo.html]: comments block without comments

**Pull request merged**